### ISSUE TITLE
Take advantage of React Query SSR in Next.js

### ISSUE THREAD

**Issue by Belco90** [octoclairvoyant/octoclairvoyant-webapp#1131]
### What feature would you like to have?

Pass initial repo and initial 10 releases data through TanStack Query SSR feature + Next.js when the comparator URL is preloaded with info.

### What problem will this feature solve?

This will make the process to fetch all the releases faster when processing a preloaded changelog.

### Do you want to help make the feature?

Yes

### Is there anything else we need to know?

This is blocked by #1127 

Useful resources:

- [TanStack Query - SSR & Next.js](https://tanstack.com/query/v4/docs/guides/ssr)

**Comment by Belco90** [comment]
Closing in favor of #860
